aws/config.go

```diff
@@ -475,6 +477,7 @@ func (c *Config) Client() (interface{}, error) {
 	awsLambdaSess := sess.Copy(&aws.Config{Endpoint: aws.String(c.LambdaEndpoint)})
 	awsKinesisSess := sess.Copy(&aws.Config{Endpoint: aws.String(c.KinesisEndpoint)})
 	awsKinesisAnalyticsSess := sess.Copy(&aws.Config{Endpoint: aws.String(c.KinesisAnalyticsEndpoint)})
+	awsKinesisAnalyticsV2Sess := sess.Copy(&aws.Config{Endpoint: aws.String(c.KinesisAnalyticsEndpoint)})
```

Comment:
I think this is okay given what the new package for v2 is supposed to be for (seems like just new API version). We can always support a separate endpoint later in the future with a conditional.